src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -127,7 +137,7 @@ public ViewVisitor readEdit(JSONObject source) throws BadInstructionsException,
                     final JSONArray argPlusType = argsAndTypes.getJSONArray(i);
                     final Object jsonArg = argPlusType.get(0);
                     final String argType = argPlusType.getString(1);
-                    methodArgs[i] = convertArgument(jsonArg, argType);
+                    methodArgs[i] = convertArgument(jsonArg, argType, assetsLoaded);
```

Comment:
In this case, I wanted to add elements to the list as a side effect, and wanted to keep the work on the stack rather than in the state of the object. This isn't great- It would be better to gather the resources separately, but there is value to having individual methods parse individual pieces of the input, so I made a trade off.